casepro/msgs/migrations/0038_no_empty_messages.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+def delete_empty_messages(apps, schema_editor):
+    Outgoing = apps.get_model('msgs', 'Outgoing')
+
+    empties = Outgoing.objects.filter(text=None)
```

Comment:
A long time ago the send message form didn't prevent sending empty messages - hence there's some of these in our db